src/java.base/unix/native/libjava/ProcessImpl_md.c

```diff
@@ -487,14 +487,14 @@ static pid_t
 spawnChild(JNIEnv *env, jobject process, ChildStuff *c, const char *helperpath) {
     pid_t resultPid;
     int i, offset, rval, bufsize, magic;
-    char *buf, buf1[16];
+    char *buf, buf1[24];
```

Comment:
Since you are here could you please increase buffer size to something safe? Max len of INT_MIN is 11 bytes, so lets have at least 3*11 + 3 bytes.

Reply:
OK, so let it be (until we get 64-bit integers and file descriptors :)

Reply:
Ok, if you let it be, I fix it in a separate RFE. Cleaner that way anyway. 

I filed https://bugs.openjdk.org/browse/JDK-8308350 and https://github.com/openjdk/jdk/pull/14045. 

Should I happen to push before you, this should be easily adaptable for you.